tensorflow/core/framework/tensor_shape.h

```diff
@@ -191,9 +191,6 @@ class TensorShapeBase : public TensorShapeRep {
   /// Appends all the dimensions from `shape`.
   void AppendShape(const TensorShapeBase& shape);
 
-  // Maximum number of dimensions in a tensor.
-  static constexpr int MaxDimensions() { return 254; }
```

Comment:
Why this deletion?

Reply:
https://github.com/fo40225/tensorflow/blob/9832df4e7acf093e2fccbb84c5362ee201ea4321/tensorflow/core/framework/tensor_shape.h#L62

this member function hide the base class implement, both return 254.